docs/tutorials/goldenpath.md

```diff
@@ -0,0 +1,497 @@
+1. Create a script called `NetworkCommandLine`.
+1. Right click on the `NetworkManager` in the hierarchy view.
+1. Create an empty GameObject underneath it.
+1. Rename it `NetworkCommandLine`.
+1. In inspector  for `NetworkCommandLine` Click **Add Component** 
+1. Click **Scripts** and add the `NetworkCommandLine.cs` script you created earlier.
+1. Open the `NetworkCommandLine.cs` script.
+1. Edit the `NetworkCommandLine.cs` script to match the following.
+
+:::tip 
+You can copy the script from here and paste it into your file.
+   1. Select the code sample.
+   1. Click **Copy** in the top right corner.
+   1. Paste it into your code editor.
+:::
+
+<details open>
+<summary>Click to show/hide the Code.
+
+</summary>
+
+``` csharp
+using System.Collections.Generic;
+using MLAPI;
+using UnityEngine;
+
+public class NetworkCommandLine : MonoBehaviour
+{
+   private NetworkManager netManager;
+
+   void Start()
+   {
+       netManager = GetComponentInParent<NetworkManager>();
+
+       if (Application.isEditor) return;
+
+       var args = GetCommandlineArgs();
+
+       if (args.TryGetValue("-mlapi", out string mlapiValue))
+       {
+           switch (mlapiValue)
+           {
+               case "server":
+                   netManager.StartServer();
+                   break;
+               case "host":
+                   netManager.StartHost();
+                   break;
+               case "client":
+         
+                   netManager.StartClient();
+                   break;
+           }
+       }
+   }
+
+   private Dictionary<string, string> GetCommandlineArgs()
+   {
+       Dictionary<string, string> argDictionary = new Dictionary<string, string>();
+
+       var args = System.Environment.GetCommandLineArgs();
+
+       for (int i = 0; i < args.Length; ++i)
+       {
+           var arg = args[i].ToLower();
+           if (arg.StartsWith("-"))
+           {
+               var value = i < args.Length - 1 ? args[i + 1].ToLower() : null;
+               value = (value?.StartsWith("-") ?? false) ? null : value;
+
+               argDictionary.Add(arg, value);
+           }
+       }
+       return argDictionary;
+   }
+}
+```
+
+</details>
+
+1. Select  **File > Build Settings > Player Settings...**
+1. Change `Full Screen` mode into `Windowed` mode.  
+1. Save your scene.
+
+   
+:::tip
+   If you are on a Pro Unity license, you may want to disable the splash screen
+:::
+
+<iframe src="https://www.youtube.com/embed/2swybHUigM8?playlist=2swybHUigM8&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+### Testing the command line helper
+1. Select **File > Build and Run**. 
+1. Create a folder called `Build`.
+1. Name the binary `GoldenPath`.  
```

Comment:
This is for Mac. On Windows it just wants the destination folder for the build. So for parity, Windows users should drop the build under the path "<path to project>Build/". Can't recall if this is different on Mac, but Unity on Windows uses the `Product Name` under Player settings as the executable name.

Reply:
have modified the text to remove the reference to two players and added the Windows instructions.